### PR TITLE
[Pipeline] adds pipeline to verify ui by executing tests

### PR DESCRIPTION
**Pull request by adamloup-enquizit**
## Description

Adds a pipeline to verify the `modernization-ui` by executing Jest tests.

### CODE REVIEW

**Comment by mpeels** [comment]
LGTM

thoughts @szamfir

File: .github/workflows/verify-modernization-ui.yaml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+jobs:
+  pipeline:
+    name: Build and verify
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0 # Shallow clones should be disabled for a better relevancy of analysis
```
Comment (mpeels) [inline .github/workflows/verify-modernization-ui.yaml:21]: image

Do we want all?

Comment (adamloup-enquizit) [reply .github/workflows/verify-modernization-ui.yaml:21]: Good question, I copied this from the sonar pipeline

Comment (szamfir) [reply .github/workflows/verify-modernization-ui.yaml]: Removing with fetch-depth is typical. You just want the latest commit

**Review by szamfir — DISMISSED** [review]
LGTM

File: .github/workflows/verify-modernization-ui.yaml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+name: Verify Modernization UI
+
+on:
+  workflow_dispatch:
+  pull_request:
+    paths:
+      - "apps/modernization-ui/**"
+
+env:
+  passed_github_token: ${{secrets.PASSED_GITHUB_TOKEN}}
```
Comment (szamfir) [inline .github/workflows/verify-modernization-ui.yaml:11]: You don't need this env. Just noticed it.

**Review by szamfir — APPROVED** [review]
LGTM

**Pull request merged**